ru/turing/courses/lesson2/animals/Human.java

```diff
@@ -0,0 +1,81 @@
+package ru.turing.courses.lesson2.animals;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Human {
+    private String date;
+    private String name;
+    private boolean sex;
+    private String adress;
+
+
+    public Human(String date, String name, boolean sex, String adress) {
+        this.date = date;
+        this.name = name;
+        this.sex = sex;
+        this.adress = adress;
+    }
+
+    public String getDate() {
+        return date;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public boolean isSex() {
+        return sex;
+    }
+
+    public void setDge(String date) {
+        this.date = date;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setSex(boolean sex) {
+        this.sex = sex;
+    }
+
+    public void setDate(String date) {
+        this.date = date;
+    }
+
+    public String getAdress() {
+        return adress;
+    }
+
+    public void setAdress(String adress) {
+        this.adress = adress;
+    }
+
+    //adress print method
+    public void adress(String adress) {
+        String[] parts = adress.split(",");
+        for (String i : parts) {
+            System.out.print(i + " ");
+        }
+    }
+
+    public void livedYearMonths(String date) {
+        SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
+        Date date1 = null;
```

Comment:
```suggestion
        Date birthDate= null;
```
Переменные лучше осмысленно называть